mautic/mautic
# Fix Encoding Issues with Special Characters in Segment Names

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #14038  



## Description







At the moment, special characters in segment names are incorrectly encoded. For example, `R&D` would become `R&#38;D` . This PR fixes that issue

---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a segment
3. Choose a name that contains `&` or `'`
4. Save the segment
5. See that the segment name is saved and displayed correctly
